website/source/docs/agent/config.html.md

```diff
@@ -217,6 +217,11 @@ configured on server nodes.
 
 ## Atlas Options
 
+**NOTE**: Nomad integration with Atlas is awaiting release of Atlas features
+for Nomad support.  Nomad validates configuration options for Atlas,
+however, Nomad does not use the configuration options for Atlas.
```

Comment:
I think we can change the second sentence to: Nomad currently only validates configuration options for Atlas but does not use them.